libvast/src/system/index.cpp

```diff
@@ -631,19 +631,22 @@ index_state::status(status_verbosity v) const {
     }
   };
   auto rs = make_status_request_state<extra_state>(self);
+  auto stats_object = record{};
+  auto sum = uint64_t{0};
+  for (const auto& [_, layout_stats] : stats.layouts)
+    sum += layout_stats.count;
+  auto xs = record{};
+  xs["total"] = count{sum};
+  stats_object["events"] = xs;
+  if (v == status_verbosity::info)
+    rs->content["statistics"] = std::move(stats_object);
```

Comment:
nit: We could also do this assignment unconditionally at the end (before the `fill_status_map` stuff). I don't have a strong opinion though so you are free to ignore this suggestion.